sklearn/ensemble/forest.py

```diff
@@ -1642,6 +1664,8 @@ class labels (multi-output problem).
     sklearn.tree.ExtraTreeClassifier : Base classifier for this ensemble.
     RandomForestClassifier : Ensemble Classifier based on trees with optimal
         splits.
+
+
```

Comment:
I would add only one separating line
```suggestion
```

Reply:
Thanks @pspachtholz , I have gone through the links and the review. I will make necessary changes and make a PR.

Thank you